src/components/Sidebar/components/NavItem/utils.spec.js

```diff
@@ -52,3 +54,65 @@ describe('hasSelectedChild', () => {
     expect(hasSelected).toBe(true);
   });
 });
+
+describe('getIcon', () => {
+  const mockDefaultIcon = 'mock-default-icon.svg';
+  const mockSelectedIcon = 'mock-selected-icon.svg';
+  const disabledIcon = <Lock />;
+
+  describe('if there is no default icon', () => {
+    it('should not return an icon', () => {
+      const icon = getIcon({});
+      expect(icon).toBeNull();
+    });
+
+    it('should not return the disabled icon', () => {
+      const icon = getIcon({
+        disabled: true
+      });
+      expect(icon).toBeNull();
+    });
+  });
+
+  describe('if there is a default icon', () => {
+    describe('if the item is disabled', () => {
+      it('should return the disabled icon', () => {
+        const icon = getIcon({
+          defaultIcon: mockDefaultIcon,
+          disabled: true
+        });
+        expect(icon).toStrictEqual(disabledIcon);
+      });
+    });
+
+    describe('if the item is not disabled', () => {
+      describe("when the item isn't selected", () => {
```

Comment:
Nit: you're switching between `if` and `when` in your descriptions.

Reply:
Yes: in my mind the `disabled` state is fixed (i.e. an item can be either disabled or not), whereas the `selected` state can be toggled on and off by the user, hence the wording difference. But I agree that we should harmonize this 🙂

Reply:
Ah, now that you've explained it, it does make sense.